easybuild/easyconfigs/p/PETSc/PETSc-3.7.3-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,32 @@
+name = 'PETSc'
+version = '3.7.2'
```

Comment:
@RvDijk this is not aligned with the name of the easyconfig...

The tests should be spotting that though, I wonder why they don't... :-/